gammapy/image/tests/test_core.py

```diff
@@ -317,6 +318,14 @@ def test_center_sky(self):
         assert_allclose(center.l.deg, self.center.l.deg, atol=1e-5)
         assert_allclose(center.b.deg, self.center.b.deg, atol=1e-5)
 
+    @requires_dependency('scipy')
+    @pytest.mark.parametrize('kernel', ['gauss', 'box', 'disk'])
+    def test_smooth(self, kernel):
+        desired = self.image.data.sum()
+        smoothed = self.image.smooth(kernel, 0.2 * u.deg)
+        actual = smoothed.data.sum()
+        assert_allclose(actual, desired)
```

Comment:
Is this a useful test?
The test image is all zero, no?
So smoothing with anything will pass the test.

A better test would depend on:
- Kernel normalisation
- Convolution behaviour at the edges
and establish the current behaviour via asserts.